### DISCUSSION TITLE
Change category of iOS app to “Social”

### DISCUSSION THREAD

**Discussion by hwittenborn** [element-hq/element-meta#2255] [Ideas]
The Element iOS app is under the `Productivity and Finance` category on my iPhone's homescreen. I feel like it'd make more sense to be under the `Social` category, which would match other apps such as Discord, Facebook Messenger and GroupMe.

**Comment by tkr-sh** [comment]
@hwittenborn element X has been tagged with "Social Network".
You can probably mark this as resolved :)
